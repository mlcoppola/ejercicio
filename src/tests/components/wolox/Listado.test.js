import React from 'react';

const { shallow } = require('enzyme');
const { Listado } = require('../../../components/wolox/Listado');

describe('Pruebas en <Listado />', () => {

    const techMock = {
        logo: 'test',
        tech: 'test',
        year: 'test',
        author: 'test',
        license: 'test',
        language: 'test',
        type: 'test'
    }
    const wrapper = shallow(<Listado tech={techMock} />);

    test('debe de mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('.listado__tecnologia').exists()).toBe(true);
    });


})
