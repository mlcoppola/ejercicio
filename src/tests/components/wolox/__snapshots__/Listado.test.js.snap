// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Listado /> debe de mostrarse correctamente 1`] = `
<div
  className="listado__tecnologia"
>
  <div
    className="listado__contenedor-imagen"
  >
    <img
      alt="test"
      src="test"
    />
  </div>
  <div
    className="listado__contenedor-info"
  >
    <p>
      Tecnología: 
      test
    </p>
    <p>
      Año: 
      test
    </p>
    <p>
      Autor: 
      test
    </p>
    <p>
      Licencia: 
      test
    </p>
    <p>
      Lenguaje: 
      test
    </p>
    <p>
      Tipo: 
      test
    </p>
  </div>
</div>
`;
